web/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+
+services:
+  testgrid:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      # args:
```

Comment:
nit: Add a comment here noting when to uncomment (e.g. `# Uncomment \`args\` to use the public TestGrid API.`)